doc/users/rpc_api.rst

```diff
@@ -7,15 +7,15 @@ The API can also be retrieved from a running service using the `api`_ and `api/s
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/app/api --cacert networkcert.pem --key user0_privk.pem --cert user0_cert.pem
+    $ curl https://<ccf-node-address>/node/api --cacert networkcert.pem --key user0_privk.pem --cert user0_cert.pem
```

Comment:
We don't need the user0 key/cert if we're calling the `node` frontend. But this is in the `users` docs, so I think we should either stick to user-auth or explicitly call out "you (or anyone else) can access the `/node` endpoints without auth.

Reply:
Sure, but the list was wrong, because getSignedIndex was always a node thing and not a common handler. So I'll go back to app and remove it then,